packages/node_modules/@node-red/nodes/core/sequence/18-sort.js

```diff
@@ -106,20 +108,21 @@ module.exports = function(RED) {
                 var key = function(msg) {
                     return ;
                 }
-                var comp = generateComparisonFunction(msg => RED.util.getMessageProperty(msg, key_prop));
+                var comp = generateComparisonFunction(mInfo => RED.util.getMessageProperty(mInfo.msg, key_prop));
                 try {
-                    msgs.sort(comp);
+                    msgInfos.sort(comp);
```

Comment:
The comparison function assumes it is sorting an array of messages. `msgInfos` is not an array of messages. The comparison function needs to be updated to compare the `.msg` property of each object in the array.

Reply:
To this purpose, I also updated the comparison function so that it can refer `.msg` property of each object in the array. 

https://github.com/node-red/node-red/blob/ee68fe063be913ac4754295882e537f8c537eb34/packages/node_modules/%40node-red/nodes/core/sequence/18-sort.js#L111